### ISSUE TITLE
configuration window does not let me enter data

### ISSUE THREAD

**Issue by UnixMonky** [cpyarger/obs-midi#125]
## Platform
Operating system and version: Arch Linux /AND/ Windows 10
OBS Studio version: 26.1.2

## Expected Behavior
Clicking on the "configure" tab of the obs-midi tool should allow me to configure the various triggers

## Current Behavior
"Channel" "Note/Control" and "Value" options cannot be adjusted.  Clicking in the fields does nothing, and the slider for "value" does not work.
"Add Mapping" button does nothing, but I suspect this is because there is not a valid config to add.

## Steps to Reproduce
1. Tools -> OBS Midi Settings -> Configure Tab
(screenshot from Linux)
![2021-04-04-170103_1025x643_scrot](https://user-images.githubusercontent.com/6799707/113521665-f422f080-9568-11eb-8b3b-f10d75b5d837.png)

## Additional information
Same behavior on both Windows 10 and Arch Linux.
Windows10 install was using the install package.
Arch was installed by downloading the .tar.gz and extracting to ~/.config

**Comment by cpyarger** [comment]
Press the listen one or listen many button and toggle a fader or knob or button

**Comment by UnixMonky** [comment]
Still not working.  Here's a video that simultaneously shows (a) me pressing the button, (b) the laptop seeing the midi data, and (c) obs-midi not reacting to it: https://youtu.be/zNuHhmktw_0

I will concede this may be something I'm doing wrong, but I have no idea what at this point.

**Comment by cpyarger** [comment]
You dont need to set the message type, and why is the listen many button pressed as well? Might the first program have taken control of the midi connection?

**Comment by UnixMonky** [comment]
Ugh, that was it.  in troubleshooting why it wasn't working the debugging grabbed the output so OBS couldn't see it.  
Closing as this isn't actually a problem.  I will say that the process is not intuitive but I understand it's still alpha.
Thanks for the help, it's greatly appreciated.

**Comment by michaljanowicz** [comment]
I have exactly same problem (Windows 10). I cannot change channel, note and value. "Add mapping" button does nothing.
Pressing "Listen One" or/and "Listen Many" changes nothing.

**Comment by michaljanowicz** [comment]
@UnixMonky  could you reopen this issue?

**Comment by UnixMonky** [comment]
I'm no longer experiencing this issue; my problems have been solved. My issues were two-fold:
1- midi device was not properly sending midi commands
2- while troubleshooting #1 I added a midi watcher to see the output; that watcher prevented obs-midi from seeing the midi commands.
Once my device was properly sending midi commands /and/ nothing else was interfering with the midi stream everything worked.  This was 100% a configuration issue on my end and not an actual obs-issue, so re-opening this ticket doesn't make sense.
